templates/web-serviceaccount.yaml

```
@@ -9,5 +9,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- if .Values.rbac.webServiceAccountAnnotations }}
+    annotations:
+{{ toYaml .Values.rbac.webServiceAccountAnnotations | indent 4 }}
+  {{- end -}}
```

```suggestion
    {{- end -}}
```
just a minor indentation